NPE in PreferenceActivity when rotating while dialog is open

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{de.danoeh.antennapod/de.danoeh.antennapod.activity.PreferenceActivity}: java.lang.NullPointerException
...
Caused by: java.lang.NullPointerException at de.danoeh.antennapod.activity.PreferenceActivity$MainFragment.onCreate(PreferenceActivity.java:105)
```

Reproduce: Open Settings, click *Update Interval or Time of Day*, rotate

--- a/app/src/main/java/de/danoeh/antennapod/activity/PreferenceActivity.java
+++ b/app/src/main/java/de/danoeh/antennapod/activity/PreferenceActivity.java
@@ -102,13 +102,17 @@ public class PreferenceActivity extends ActionBarActivity {
         public void onCreate(Bundle savedInstanceState) {
             super.onCreate(savedInstanceState);
             addPreferencesFromResource(R.xml.preferences);
-            instance.preferenceController.onCreate();
+            if(instance.preferenceController != null) {
+                instance.preferenceController.onCreate();
+            }
         }
 
         @Override
         public void onResume() {
             super.onResume();
-            instance.preferenceController.onResume();
+            if(instance.preferenceController != null) {
+                instance.preferenceController.onResume();
+            }
         }
     }
 }

